learning/web/src/pages/index.js

```diff
@@ -11,7 +11,7 @@ import Link from "../components/Link"
 
 const HomepageLink = styled.div`
   a {
-    color: ${anchorColour};
+    color: #1a6db5;
```

Comment:
might wanna bring this into a constant later

Reply:
this was a small colour tweak for the appropriate colour contrast as was failing the pa11y check. We can revisit this later.